n3fit/src/evolven3fit/evolven3fit_API.py

```diff
@@ -0,0 +1,209 @@
+from ekobox import gen_theory, gen_op, genpdf, gen_info
+import pathlib
+import numpy as np
+import yaml
+from . import utils
+from validphys.loader import Loader
+import os
+from ekomark import apply
+from eko import basis_rotation as br
+from eko import run_dglap
+from eko import output
+import logging
+
+logger = logging.getLogger(__name__) 
+
+def evolve_fit(conf_folder, op_card_dict, t_card_dict ,eko_path=None, dump_eko=None):
+    """
+    Evolves all the fitted replica in conf_folder/nnfit 
+
+    Parameters
+    ----------
+
+        conf_folder: str or pathlib.Path
+            path to the folder containing the fit
+        op_card_dict: dict
+            user settings for the op_card
+        t_card_dict: dict
+            user settings for the t_card
+        eko_path: str or pathlib.Path
+            path where the eko is stored (if None the eko will be
+            recomputed)
+        dump_eko: str or pathlib.Path
+            path where the eko is dumped (if None the eko won't be
+            stored)
+    """
+    log_file = pathlib.Path(conf_folder) / f"evolven3fit.log"
+    if log_file.exists():
+        log_file.unlink()
```

Comment:
If the log file already exist, maybe we want to error out `evolven3fit has already been run`?